packages/maskbook/src/plugins/PluginUI.ts

```diff
@@ -26,12 +27,13 @@ sideEffect.then(() => {
     plugins.add(RedPacketPluginDefine)
     plugins.add(FileServicePluginDefine)
     plugins.add(ITO_PluginDefine)
-    plugins.add(NFT_PluginsDefine)
+    // plugins.add(NFTPluginsDefine)
```

Comment:
So this is going to remove NFT plugin? cc @guanbinrui

Reply:
Will add back if the plugin on/off function is ready.

Reply:
why close it for now?